docs/pages/desktop-access/introduction.mdx

```diff
@@ -1,44 +1,72 @@
 ---
-title: Desktop Access
-description: Teleport desktop access introduction and resources.
+title: Manage Windows resources
+description: Demonstrates how you can manage access to Windows desktops with Teleport.
 videoBanner: n2h0GisWdss
 ---
 
-Teleport manages graphical desktop access to remote hosts. With Teleport, you
-get:
+The topics in this guide describe how to configure Teleport to provide secure, passwordless 
+access to Microsoft Windows desktops and servers. Teleport for Windows desktops provides the
+following key features:
 
-- A password-less login experience backed by strong cryptographic
-  authentication.
-- Role-based access control (RBAC) for groups of hosts and users.
-- Support for copying and pasting to and from remote hosts.
-- Audit log and recording of all desktop connections.
+- Passwordless access to Windows hosts backed by secure cryptographic authentication.
+- Configurable role-based access controls (RBAC) for groups of hosts and users.
+- Configurable clipboard and directory sharing for copying and pasting to and from 
+remote Windows hosts.
+- Session recording for all desktop activity.
+- Audit logs that track user activity.
 
-<Admonition
-  type="note"
-  title="Supported platforms"
->
-  Only Windows hosts accessible over RDP are supported. Specifically:
+Teleport Desktop Access relies on the remote desktop protocol (RDP) to connect to
+remote Windows hosts. Therefore, Teleport Desktop Access only supports the following 
+Windows hosts that are accessible over RDP:
 
-| Teleport Version              | Windows Desktop                                           | Active Directory |
-|-------------------------------|-----------------------------------------------------------|------------------|
-| Open source Teleport          | Windows Server 2012 R2 / Windows 10 or newer              | Required         |
-| Cloud and Enterprise Teleport | Windows Server 2012 R2 / Windows 10 or newer              | Optional         |
+- Windows Server 2012 R2, or later.
+- Windows 10, or later.
 
-</Admonition>
+The following diagram provides a simplified view of the architecture for Teleport
+Desktop Access:
+
+```mermaid
```

Comment:
Aren't we trying to get away from mermaid diagrams?

Reply:
@ptgott, are we? I think they're kind of ugly but I thought they were recommended.

Reply:
We are. I thought I'd let this one pass because it does take a while to create Lucidchart diagrams, but if you have the time, we should make this a Lucidchart diagram.

Reply:
I don't have access to the Lucidchart library mentioned in the style guide.